src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -465,6 +496,24 @@ public int countApplicationUpdatedMessages() {
         return messageOccurrences;
     }
 
+    /**
+     * Try to get log file from server directory first.
+     * If the server directory path cannot be resolved, get log file from the server task instead.
+     * 
+     * @param serverTask the server task
+     * @return the messages log file for the server
+     */
+    private File getMessagesLogFile(ServerTask serverTask) {
+        File logFile;
+        try {
+            String logsDirectory = serverDirectory.getCanonicalPath() + "/logs";
+            logFile = new File(logsDirectory, "messages.log");
```

Comment:
I think you can do this instead and avoid calling getCanonicalPath(): `logFile = new File(serverDirectory, "/logs/messages.log");`

Reply:
done